db/migrate/20150106050733_set_mysql_to_unicode_mb4.rb

```diff
@@ -0,0 +1,38 @@
+class SetMysqlToUnicodeMb4 < ActiveRecord::Migration
+
+  UTF8_PAIRS = {
+    'comments': 'text',
+    'messages': 'text',
+    'poll_answers': 'answer',
+    'polls': 'question',
+    'posts': 'text',
+  }
+
+  def self.up
+    if ENV['DB'] == 'mysql'
```

Comment:
Actually I just remembered we already have https://github.com/diaspora/diaspora/blob/develop/lib/configuration_methods.rb#L121 which can be called through `AppConfig.postgres?`. Don't need that often luckily!